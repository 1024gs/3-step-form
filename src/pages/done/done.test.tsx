import React from '../../../vendors/react';
import {shallow} from 'enzyme';
import DonePage from './done';

describe('DonePage', () => {
  it('renders', () => {
    const wrapper = shallow(<DonePage />);
    expect(wrapper).toMatchSnapshot();
  });
});
