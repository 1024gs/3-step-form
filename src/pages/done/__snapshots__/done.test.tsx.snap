// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DonePage renders 1`] = `
<div
  className="app"
>
  <Navbar />
  <br />
  <div
    className="container"
  >
    <FormNav />
    <br />
    <h4
      style={
        Object {
          "textAlign": "center",
        }
      }
    >
      Well done!
    </h4>
    <p
      style={
        Object {
          "textAlign": "center",
        }
      }
    >
      Please verify your email address, you should have received an email from us already!
    </p>
  </div>
</div>
`;
